q-gate.js

```diff
@@ -32,6 +32,9 @@ module.exports = function(RED) {
         this.triggerCmd = config.triggerCmd.toLowerCase();
         this.flushCmd = config.flushCmd.toLowerCase();
         this.resetCmd = config.resetCmd.toLowerCase();
+        this.peekCmd = (config.peekCmd || "peek").toLowerCase();
```

Comment:
I'm not sure why the logical OR (||) is needed. The new commands should pick up values from the `default` object if needed, just like the old ones.

Reply:
When I updated the node and then restarted node-red the new fields were not filled in but were left empty. I had to put the OR in in order to stop it showing an error.  When I opened the node it showed the new fields with the default values, but grey as they hadn't actually been entered.

Reply:
OK. I'll sort this out in testing.